test/fixtures/shortcuts/glue-json-table-defaults.json

```diff
@@ -0,0 +1,41 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Metadata": {},
+  "Parameters": {},
+  "Mappings": {},
+  "Conditions": {},
+  "Resources": {
+    "MyTable": {
+      "Type": "AWS::Glue::Table",
+      "Properties": {
+        "CatalogId": {
+          "Ref": "AWS::AccountId"
+        },
+        "DatabaseName": "my_database",
+        "TableInput": {
+          "Description": {
+            "Fn::Sub": "Created by the ${AWS::StackName} CloudFormation stack"
+          },
+          "Name": "my_table",
+          "Parameters": {
+            "classification": "json"
+          },
+          "PartitionKeys": [],
+          "TableType": "EXTERNAL_TABLE",
+          "StorageDescriptor": {
+            "Compressed": false,
+            "InputFormat": "org.apache.hadoop.mapred.TextInputFormat",
+            "Location": "s3://fake/location",
+            "NumberOfBuckets": 0,
```

Comment:
I would expect `NumberOfBuckets` to be absent in the default case and only be present (and `>0` I guess?) if there are `BucketColumns`.